Entwurf/Dokumentation.tex

```diff
@@ -0,0 +1,95 @@
+\documentclass[10pt,a4paper]{article}
+\usepackage[utf8]{inputenc}
+\usepackage[german]{babel}
+\usepackage{graphicx}
+\usepackage[table,x11names]{xcolor}
+
+\begin{document}
+
+Die Datenbank für die Applikation besteht aus vier Tabellen : Group, Reservation, Appointment und Booking.
+Die Struktur der Tabellen ist in Abbildung \ref{fig:ER-Diagram} dargestellt.
```

Comment:
Das `\ref` funktioniert nur, wenn du in der figure auch ein `\label{fig:ER-Diagramm}` hast.